src/x11/ewmh.cpp

```diff
@@ -77,6 +77,13 @@ namespace ewmh_util {
     return desktop;
   }
 
+  unsigned int get_number_of_desktops(int screen) {
+    auto conn = initialize().get();
+    unsigned int desktops = XCB_NONE;
```

Comment:
```suggestion
    uint32_t desktops = 0;
```

Reply:
Fair enough, we don't really need to do this here. I added an entry in our refactoring project to use fixed width integers for everything that interacts with EWMH since the EWMH spec defines the bit-width of the numbers.